BUILD_ME.md

```diff
@@ -13,95 +13,305 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-
-http://hpccsystems.com
+https://hpccsystems.com
 ```
 
-To build for Linux:
--------------------
+# Build Instructions
 
-Prerequisites: (w/ base Ubuntu 14)
-```
-sudo apt-get install cmake
-sudo apt-get install bison
-sudo apt-get install flex
-sudo apt-get install binutils-dev
-sudo apt-get install libiberty-dev
-sudo apt-get install slapd
-sudo apt-get install openldap-dev
-sudo apt-get install libicu-dev
-sudo apt-get install libxslt-dev
-sudo apt-get install zlib1g-dev
-sudo apt-get install libarchive-dev
-sudo apt-get install libboost-all-dev
-sudo apt-get install libssl-dev
-sudo apt-get install libapr1-dev
-sudo apt-get install libaprutil1-dev
-sudo apt-get install clang
-sudo apt-get install libtool
-sudo apt-get install autotools-dev
-sudo apt-get install automake
-```
+* Visit [https://github.com/hpcc-systems/HPCC-Platform/wiki](https://github.com/hpcc-systems/HPCC-Platform/wiki) for upto date help and guidance.
```

Comment:
minor: should be 'up to date' I think.

Reply:
I'm not sure why this confused me so much.  I've learnt something new here: https://www.lexico.com/definition/up_to_date.  Apparently, compound adjectives should be hyphenated if used before the noun but not if it's used after!